packages/nodejs/src/net/UdpChannelNode.ts

```diff
@@ -129,7 +130,10 @@ export class UdpChannelNode implements UdpChannel {
         return new Promise<void>((resolve, reject) => {
             this.socket.send(data, port, host, error => {
                 if (error !== null) {
-                    const netError = new NetworkError(error.message);
+                    const netError =
+                        error instanceof Error && error.message.includes("EHOSTUNREACH")
```

Comment:
Testing error.code === "EHOSTUNREACH" seems cleaner than searching the error message

Reply:
good idea ... 

I decided for 

> "code" in error && error.code === "EHOSTUNREACH"

for simplicity ... alternatively I would have needed to check how to get the correct typing for this special Nodejs ErrnoException type